appveyor.yml

```diff
@@ -0,0 +1,23 @@
+environment:
+  matrix:
+    - nodejs_version: '6'
+    - nodejs_version: '4'
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - set CI=true
+  - set PATH=%APPDATA%\npm;%PATH%
+  - npm install
+
+matrix:
+  fast_finish: true
+
+build: off
+shallow_clone: true
+
+test_script:
```

Comment:
The test script and env vars also needs to keep in sync with `.travis.yml`. Can add comments in both places asking to keep them in sync with one another?